app/components/contents_component.rb

```diff
@@ -21,4 +21,8 @@ def upload_csv?
   def download_csv?
     !user_version_view?
   end
+
+  def structural_link_path
+    user_version_view? ? item_user_version_structure_path(@document.id, @view_token) : item_structure_path(@view_token)
+  end
```

Comment:
@jcoyne I think you made the initial change a couple years ago where we're passing an encrypted token to path helpers (to allow us to lazy-load turbo frames). At the time, the token was used in place of an item ID. Now we have user version-aware path helpers that require two arguments, an item ID and a user version ID, and that makes me wonder what the args to the path helper ought to be. Can you advise? E.g., should both args be a token? the same token or diff tokens? should just one arg be a token? if so, which one, and why? I'm not sure any of us have a better handle on this than you (but am happy to be wrong about that), and I think we ought to understand it more before we proceed.

Reply:
The purpose of the token was to make the assurance that the token holder had access to view the resource, not only to be an identifier of the resource.  This allowed us to just show the resource without having to authorize it again.